src/sentry/static/sentry/app/views/groupDetails/organization/groupTagValues.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import GroupTagValues from '../shared/groupTagValues';
+
+export default class OrganizationGroupTagValues extends React.Component {
+  render() {
+    const query = {};
+    return <GroupTagValues {...this.props} query={query} />;
```

Comment:
How will the environment selection filter its way down to here?

Reply:
The entire query object is passed down via the `query` prop which allows the org/project variants to do different types of filtering.